src/Microsoft.Crank.Jobs.Bombardier/Program.cs

```diff
@@ -62,17 +57,44 @@ static async Task Main(string[] args)
 
             args = argsList.ToArray();
 
-            var bombardierUrl = _bombardierUrls[Environment.OSVersion.Platform];
+            string bombardierUrl = null;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                bombardierUrl = "https://github.com/codesenberg/bombardier/releases/download/v1.2.4/bombardier-windows-amd64.exe";
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                bombardierUrl = "https://github.com/codesenberg/bombardier/releases/download/v1.2.4/bombardier-linux-amd64";
+            }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                bombardierUrl = "https://github.com/codesenberg/bombardier/releases/download/v1.2.4/bombardier-darwin-amd64";
+            }
+            else
+            {
+                Console.WriteLine("Unsupported platform");
+                return;
+            }
+
             var bombardierFileName = Path.GetFileName(bombardierUrl);
 
+            Console.WriteLine($"Downloading bombardier from {bombardierUrl} to {bombardierFileName}");
             using (var downloadStream = await _httpClient.GetStreamAsync(bombardierUrl))
             using (var fileStream = File.Create(bombardierFileName))
             {
                 await downloadStream.CopyToAsync(fileStream);
-                if (Environment.OSVersion.Platform == PlatformID.Unix)
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ||
+                    RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
                 {
+                    Console.WriteLine($"Setting execute permission on exeecutable {bombardierFileName}");
```

Comment:
```suggestion
                    Console.WriteLine($"Setting execute permission on executable {bombardierFileName}");
```

I would have changed it myself, by apparently you revoked the PR permissions (the default is to allow PRs to be edited by the owner of the targetted repo)

Reply:
Hmm, that's weird. I didn't get that question I think. Maybe that's because this is coming from a repo in another org? I'll check that. Suggested change looks good though, so I'll just commit it.